students/david_baylor/lesson3/slicing.py

```diff
@@ -0,0 +1,48 @@
+"""
+slicing.py
+By David Baylor on 10/10/19
+uses python 3
+
+Demonstrates basic slicing.
+"""
+
+def exchange_first_last(seq):
+    """exchange the first last item of the sequence"""
+    first = seq[:1]
+    last = seq[-1:]
+    seq[:1], seq[-1:] = last, first
+    return seq
+
+def remove_every_other(seq):
+    """remove every other item in the sequence"""
+    new_list = []
+    for i in range(len(seq)):
+        if not i % 2:
+            new_list.append(seq[i])
+    return new_list
+
+def remove_first_last_4(seq):
+    """remove the first and last 4 items in the lsit"""
+    del seq[:4]
+    del seq[-4:]
+    return seq
```

Comment:
you're kind of using slicing here

but the expected here is this `return seq[4:-4:2]`